lib/flagutil/array.go

```diff
@@ -10,6 +10,8 @@ import (
 // NewArray returns new Array with the given name and description.
 func NewArray(name, description string) *Array {
 	var a Array
+	description += "\nSupports `array` of values separated by comma" +
```

Comment:
I believe there are array flags in the repository with manually specified similar description. Could you inspect all the `flag
Array` vars in the repository and remove the similar text there?

Reply:
Verified all the uses of `flagutil.NewArray` and didn't found cases which may be changed.